cmd/backup-manager/app/export/manager.go

```diff
@@ -63,12 +63,7 @@ func (bm *BackupManager) ProcessBackup() error {
 
 	var db *sql.DB
 	err = wait.PollImmediate(constants.PollInterval, constants.CheckTimeout, func() (done bool, err error) {
-		db, err = util.OpenDB(bm.getDSN(constants.TidbMetaDB))
-		if err != nil {
-			klog.Warningf("can't open connection to tidb cluster %s, err: %v", bm, err)
-			return false, nil
-		}
-
```

Comment:
Why remove the error check here?

Reply:
OK. util.OpenDB already has db.Ping logic. I've moved the redundant  `db.Ping`.